app/src/main/AndroidManifest.xml

```diff
@@ -77,6 +77,10 @@
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value="org.fossasia.susi.ai.login.LoginActivity" />
         </activity>
+        <activity
+            android:name=".chat.YouTubeActivity"
+            android:theme="@style/Theme.AppCompat.Dialog">
+        </activity>
```

Comment:
Check if this is going to create problems in fdroid version

Reply:
@iamareebjamal there is no problem building the fdroid variant so I think the build process in fdroid should pass easily

Reply:
Try running it

Reply:
@iamareebjamal it is running